config/prod.toml

```diff
@@ -61,7 +61,7 @@
             caPem              = ""
 
 [cache]
-    driver                    = "consul"
+    driver                    = "redis"
```

Comment:
shouldn't you also define the redisconfig with the drive

Reply:
This is being defined via kubestash. Please refer [https://razorpay.atlassian.net/browse/BAD-7487](https://razorpay.atlassian.net/browse/BAD-7487)
The base config is in default.yaml